### ISSUE TITLE
Write converter

### ISSUE THREAD

**Issue by RLC-DCPPC** [RLC-DCPPC/NeMO-Analytics#4]
- [ ] Need a better description and an estimate (0,1/2,1,2,3,5,8,13,20,40,100).

- [x] Assign to carlo and apaala

**Comment by seth-ament** [comment]
This task is dependent on the completion of metadata standards for NeMO (NEMO-57) and is closely related to NEMO-56 (mapping of these same non-BICCN datasets to BCDC standards.

**Comment by RLC-DCPPC** [comment]
converter between metadata for data required by NeMO and connect so gEAR can read directly.

**Comment by apaala** [comment]
@seth-ament are NEMO-57 and NEMO-56 Jira tickets?

**Comment by seth-ament** [comment]
Yes. The NEMO Archive project is using Jira for task management. Do you have access to the NEMO board?

**Comment by apaala** [comment]
No I dont, I will put in a Jira ticket to be added to it.

**Comment by RLC-DCPPC** [comment]
Kamino cloned this issue to [nemoarchive/analytics](https://github.com/nemoarchive/analytics/issues/6)
